mobile_verifier/migrations/35_verified_subscriber_verified_mapping_event.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS verified_subscriber_verified_mapping_event (
```

Comment:
NIT: table doesn't need the `verified` prefix since all records in the table will be verified.